src/data_info/services/stat.clj

```diff
@@ -121,21 +127,23 @@
       (cset/intersection all-keys (cset/difference includes-set excludes-set))))
 
 (defn ^IPersistentMap path-stat
-  [^IPersistentMap cm ^String user ^String path & {:keys [filter-include filter-exclude] :or {filter-include nil filter-exclude nil}}]
-  (let [path (ft/rm-last-slash path)
-        included-keys (process-filters filter-include filter-exclude)]
-    (log/debug "[path-stat] user:" user "path:" path)
-    (validators/path-exists cm path)
-    (let [base-stat (if (needs-any-key? included-keys :type :date-created :date-modified :file-size :md5)
-                      (info/stat cm path)
-                      {:path path})]
-      (decorate-stat cm user base-stat included-keys))))
+  [^IPersistentMap cm ^String user ^String path & {:keys [filter-include filter-exclude validate?] :or {filter-include nil filter-exclude nil validate? true}}]
```

Comment:
in no particular place, all the changes in this file are potentially useful but don't actually affect the finished manifest endpoint. The prior code used stats, so I optimized them as I went along, but ended up not using it at all in the end. Might help elsewhere, maybe